<?xml version="1.0"?>
<launch>

  <arg name="robot_sim"                     default="true"/>
  <arg name="camera_sim"                    default="true"/>

  <arg name="use_sdh"                       default="false"/>
  <arg name="use_iiwa"                      default="false"/>
  <arg name="use_interbotix"                default="true"/>

  <!-- Model of the iiwa to use : iiwa7, iiwa14 -->
  <arg name="robot_model"                   default="px150"/>
  <arg name="rviz"                          default="true" />

  <!-- The is gonna have its nodes/topics under a namespace with the same name. -->
  <arg name="robot_name"                    default="px150"/> <!-- $(arg robot_model) -->
  <arg name="camera_name"                   default="camera"/>

  <arg name="use_calibration"		    default="true"/>

  <include file="$(find flex_grasp)/launch/enviroment.launch">
    <arg name="robot_sim"               value="$(arg robot_sim)"/>
    <arg name="camera_sim"              value="$(arg camera_sim)"/>

    <arg name="use_interbotix"          value="$(arg use_interbotix)"/>
    <arg name="use_sdh"                 value="$(arg use_sdh)"/>
    <arg name="use_iiwa"                value="$(arg use_iiwa)"/>

    <arg name="robot_model"             value="$(arg robot_model)"/>
    <arg name="robot_name"              value="$(arg robot_name)"/>
    <arg name="camera_name"             value="$(arg camera_name)"/>

    <arg name="rviz"                    value="$(arg rviz)"/>
    <arg name="use_calibration"		value="$(arg use_calibration)"/>
  </include>

</launch>
